### ISSUE TITLE
Allow overriding default branch

### ISSUE THREAD

**Issue by jayvdb** [lorinkoz/django-unmigrate#29]
I have a repo with 'develop' as the default branch.  The logic in `get_main_branch` looks for `main` or `master`.

It would be nice to have an env var or setting which I can use to override the default branch.

**Comment by lorinkoz** [comment]
Agreed, I had it as a Django setting at some point, but it felt overkill to pollute Django with Git branch information. However, given that the common options for main branch names are not so many, I wonder if we should just add some additional static values instead. Wdyt?

**Comment by jayvdb** [comment]
I don't think 'develop' is very common, but it is common enough, but I've seen repos with both master/main and develop, where develop is the stable unreleased and would be what should be used by django-unmigrate.  I agree that it is a bit odd to have a git branch setting in Django, and an envvar is a bit silly.

It is possible to get the default branch name from GitHub, and shove that into a section of .git/config, and that is probably possible with GitLab also.

To begin with django-unmigrate could use a section in .git/config, and later on add the feature to automatically populate it.
